jcl/src/java.management/share/classes/java/lang/management/LockInfo.java

```diff
@@ -145,7 +146,7 @@ public static LockInfo from(CompositeData compositeData) {
 	 * The string will hold both the name of the lock object's class and it's
 	 * identity hash code expressed as an unsigned hexadecimal. i.e.<br>
 	 * <p>
-	 * {@link #getClassName()}&nbsp;+&nbsp;&commat;&nbsp;+&nbsp;Integer.toHexString({@link #getIdentityHashCode()})
+	 * {@link #getClassName()} &nbsp;+&nbsp;&nbsp;+&nbsp;Integer.toHexString({@link #getIdentityHashCode()})
```

Comment:
Was it intentional to remove the `&comma`?

Reply:
No, have not, let me try this one replacing. thanks

Reply:
@keithc-ca  thanks it worked :)